src/AccessibilityInsights.SharedUx/Controls/SettingsTabs/AboutTabControl.xaml.cs

```diff
@@ -21,7 +23,8 @@ public partial class AboutTabControl : UserControl
         public AboutTabControl()
         {
             InitializeComponent();
-            lbVersion.Content = VersionTools.GetAppVersion();
+            InitializeVersionInformationLink();
```

Comment:
>InitializeVersionInformationLink [](start = 12, length = 32)

@Jalkire, I'd love to bind these in XAML rather than in code, but my attempts were unsuccessful. I welcome any suggestions you may have

Reply:
I think `ChannelConfigControl` and its accompanying `ChannelConfigViewModel` should be one of the simpler uses of bindings that you could model this after. One potentially unintuitive detail of bindings is that the property to which you are binding needs to be a member of the UIElement's [DataContext](https://docs.microsoft.com/en-us/dotnet/api/system.windows.frameworkelement.datacontext?view=netframework-4.8).

A route you could take is:
1. Make some `AboutTabViewModel` class which will contain the dynamic values you want to display
2. Set it as `AboutTabControl`'s DataContext
3. Bind the appropriate properties

[Official example](https://docs.microsoft.com/en-us/dotnet/framework/wpf/data/how-to-specify-the-binding-source?view=netframework-4.8)

Reply:
Thanks for the help here! Binding added in iteration 4

---
In reply to: [294556168](https://github.com/microsoft/accessibility-insights-windows/pull/388#discussion_r294556168) [](ancestors = 294556168)

Reply:
Iteration 4 uses binding

---
In reply to: [294487918](https://github.com/microsoft/accessibility-insights-windows/pull/388#discussion_r294487918) [](ancestors = 294487918)